configs.go

```diff
@@ -55,3 +60,48 @@ func GetDefaultConf() PoolConfig {
 		MinConnPoolSize: 0,
 	}
 }
+
+// GetDefaultSSLConfig return the default tls.Config with path
```

Comment:
`// GetDefaultSSLConfig reads the files in the given path and returns a tls.Config object`

Reply:
done